addOns/retest/src/main/java/org/zaproxy/addon/retest/EditAlertPanel.java

```diff
@@ -0,0 +1,306 @@
+import javax.swing.DefaultComboBoxModel;
+import javax.swing.JComboBox;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JSpinner;
+import javax.swing.JTextField;
+import javax.swing.ScrollPaneConstants;
+import javax.swing.border.TitledBorder;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jdesktop.swingx.JXPanel;
+import org.jdesktop.swingx.ScrollableSizeHint;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.core.scanner.Alert;
+import org.parosproxy.paros.extension.AbstractPanel;
+import org.parosproxy.paros.network.HttpRequestHeader;
+import org.zaproxy.zap.model.Vulnerabilities;
+import org.zaproxy.zap.model.Vulnerability;
+import org.zaproxy.zap.utils.FontUtils;
+import org.zaproxy.zap.utils.ZapNumberSpinner;
+import org.zaproxy.zap.utils.ZapTextArea;
+import org.zaproxy.zap.utils.ZapTextField;
+import org.zaproxy.zap.view.LayoutHelper;
+
+public class EditAlertPanel extends AbstractPanel {
+
+    private static final long serialVersionUID = 1L;
+    private static final Logger logger = LogManager.getLogger(EditAlertPanel.class);
+    private static final Insets DEFAULT_INSETS = new Insets(1, 1, 1, 1);
+
+    private JScrollPane alertPane = null;
+    private ZapTextArea defaultOutput = null;
+    private JXPanel alertDisplay = null;
+    private CardLayout cardLayout = null;
+
+    private ZapNumberSpinner alertEditPlugin = null;
+    private JComboBox<String> alertEditName = null;
+    private ZapTextField alertEditUrl = null;
+    private JComboBox<String> alertEditMethod = null;
+    private ZapTextField alertEditAttack = null;
+    private JComboBox<String> alertEditParam = null;
+    private ZapTextField alertEditEvidence = null;
+    private JComboBox<String> alertEditConfidence = null;
+    private JComboBox<String> alertEditRisk = null;
+    private ZapTextArea alertOtherInfo = null;
+    private DefaultComboBoxModel<String> methodListModel = null;
+    private DefaultComboBoxModel<String> nameListModel = null;
+    private DefaultComboBoxModel<String> paramListModel = null;
+
+    private List<Vulnerability> vulnerabilities = null;
+
+    public EditAlertPanel() {
+        super();
+        initialize();
+    }
+
+    private void initialize() {
+        cardLayout = new CardLayout();
+        this.setLayout(cardLayout);
+        this.setName("AlertView");
+        this.add(getAlertPane(), getAlertPane().getName());
+    }
+
+    private JScrollPane getAlertPane() {
+        if (alertPane == null) {
+            alertPane = new JScrollPane();
+            alertPane.setHorizontalScrollBarPolicy(ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER);
+            alertPane.setViewportView(getAlertDisplay());
+            alertPane.setName("alertPane");
+        }
+        return alertPane;
+    }
+
+    private ZapTextArea createZapTextArea() {
+        ZapTextArea ZapTextArea = new ZapTextArea(3, 30);
```

Comment:
```suggestion
        ZapTextArea zapTextArea = new ZapTextArea(3, 30);
```

Should be named with proper camelCase.